suite-native/module-settings/src/tests/slice.test.ts

```diff
@@ -0,0 +1,37 @@
+import { configureStore, Store } from '@reduxjs/toolkit';
+
+import { fiatCurrencies } from '@suite-common/suite-config';
+
+import { appSettingsReducer, setColorScheme, setFiatCurrency } from '../slice';
+
+const store: Store = configureStore({
+    reducer: {
+        appSettings: appSettingsReducer,
+    },
+});
+
+describe('change color scheme', () => {
+    const state = store.getState().appSettings;
+
+    it('loads initial state correctly', () => {
+        expect(state.colorScheme).toEqual('system');
```

Comment:
this test is seems kind of useless

Reply:
[9ec89f2](https://github.com/trezor/trezor-suite/pull/5895/commits/9ec89f267e1597900d345e924b82975547d8cf51)

Reply:
still here